set.mm

```diff
@@ -55283,8 +55284,23 @@ of the restricted converse (see ~ cnvrescnv ).  (Contributed by NM,
     optocl.2 $e |- ( <. x , y >. = A -> ( ph <-> ps ) ) $.
     optocl.3 $e |- ( ( x e. B /\ y e. C ) -> ph ) $.
     $( Implicit substitution of class for ordered pair.  (Contributed by NM,
-       5-Mar-1995.) $)
+       5-Mar-1995.)  Shorten and reduce axiom usage.  (Revised by TM,
+       29-Dec-2025.) $)
     optocl $p |- ( A e. D -> ps ) $=
+      ( cxp wcel cv cop wceq wa wex elxpi wb eqcoms imbitrid imp exlimivv syl
+      eleq2s ) BEFGLZHEUGMECNZDNZOZPZUHFMUIGMQZQZDRCRBCDEFGSUMBCDUKULBULAUKBKAB
+      TUJEJUAUBUCUDUEIUF $.
+  $}
```

Comment:
The new and OLD version share the same hypotheses.  So the file can be kept shorter by deleting line 55293 and lines 55295 to 55299.